blazor/rich-text-editor/code-snippet/mention-integration.razor

```diff
@@ -0,0 +1,102 @@
+@using Syncfusion.Blazor.DropDowns
+@using Syncfusion.Blazor.RichTextEditor
+
+
+<SfRichTextEditor ID="mentionIntegration">
+    <p>Hello 
+    <span contenteditable="false" class="e-mention-chip"><a title="maria@gmail.com">@@Maria</a></span>
+    </p>
+    <p>Welcome to the mention integration with rich text editor demo. Type <code>@@</code> character and tag user from the suggestion list. </p>
+</SfRichTextEditor>
+
+<div id="mention_integration">
+    <SfMention TItem="PersonData" Target="#mentionIntegration .e-rte-content .e-content" DataSource="@EmailData" SuggestionCount=8 AllowSpaces="true" PopupHeight="200px" PopupWidth="250px">
```

Comment:
You are not used the ID to target. @Kokila13081999  check content

Reply:
Mapped the content editable ID to mention Target.